src/app/dashboard/dashboard.component.html

```diff
@@ -1,49 +1,46 @@
 <!DOCTYPE html>
```

Comment:
lines 1-7 aren't needed here since we are putting this in the app.component.html (we discovered this last night)